src/App.jsx

```diff
@@ -54,6 +55,12 @@ function App() {
 	const [stakeAmount] = useDebounce(stakeInput, 500.0);
 	const [apiConnected, setApiConnected] = React.useState(false);
 	const [isLoaded, setIsLoaded] = React.useState(false);
+	let [validators, setValidators] = React.useState([{name: 'None', stashId: "", amount: 0, risk: 0.00}]);
```

Comment:
Why is state not `const`?

Reply:
previously, I was re-assigning validators, but have changed it now.